tools/lib/test_server.py

```diff
@@ -16,7 +17,11 @@
 except ImportError as e:
     print("ImportError: {}".format(e))
     print("You need to run the Zulip tests inside a Zulip dev environment.")
-    print("If you are using Vagrant, you can `vagrant ssh` to enter the Vagrant guest.")
+    username = getpass.getuser()
```

Comment:
How hard would it be to use `check_venv` from `sanity_check.py` in `test_server.py`?